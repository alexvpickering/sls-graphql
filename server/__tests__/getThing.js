import axios from 'axios';
import gql from 'graphql-tag';

it('works', async () => {
  return axios
    .post('http://localhost:4000/graphql', {
      query: gql`
        {
          getThing(id: 2) {
            two
          }
        }
      `,
    })
    .then(res => {
      expect(res.data.data).toMatchSnapshot();
    });
});
