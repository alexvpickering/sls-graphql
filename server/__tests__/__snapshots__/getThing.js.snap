// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
Object {
  "getThing": Object {
    "two": "thing 2 because",
  },
}
`;
